dvc/schema.py

```diff
@@ -77,6 +83,7 @@
         Any(str, OUT_PSTAGE_DETAILED_SCHEMA)
     ],
     Optional(StageParams.PARAM_PLOTS): [Any(str, PLOT_PSTAGE_SCHEMA)],
+    Optional(StageParams.PARAM_DVCLIVE): [Any(str, DVCLIVE_PSTAGE_SCHEMA)],
```

Comment:
Since there can only be 1 dvclive entry, maybe we could limit it in schema too in addition to your assert?